src/main/webapp/resources/js/pages/cart/components/share/shareSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createAction, createSlice } from "@reduxjs/toolkit";
+
```

Comment:
File comments

Reply:
Updated in 797d2b7c89f081d26239eb59203d53f3613d49ce